style.css

```
@@ -0,0 +1,1381 @@
+  height: 50px;
+  line-height: 50px;
+  color: white;
+  text-decoration: none;
+  background: black;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.border-button:hover {
+  border-width: 5px;
+}
+
+.text1 {
+  position: absolute;
+  left: 16px;
+  top: 70px;
+  padding-left: 10px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.text2 {
+  position: absolute;
+  width: 320px;
+  height: 126px;
+  left: 16px;
+  top: 224px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+}
+
+.rectangle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  top: 280px;
+  margin: 0 auto;
+  width: 340px;
+  height: 50px;
+  color: #fdfdfd;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  background: #0e0e0f;
+}
+
+.text3 {
+  position: absolute;
+  width: 181px;
+  left: 16px;
+  top: 680px;
+  padding-left: 10px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.projectpageimage1 {
+  background: url("Rectangle_21.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 206px;
+  top: 500px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage2 {
+  background: url("second_project.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 206px;
+  top: 414px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage3 {
+  background: url("thirdproject.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 206px;
+  top: 328px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage4 {
+  background: url("fourthproject.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 209px;
+  top: 241px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage5 {
+  background: url("fifthproject.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 206px;
+  top: 153px;
+  position: relative;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+}
+
+.projectpageimage6 {
+  background: url("sixthproject.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  width: 100%;
+  height: 206px;
+  top: 73px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
```

To make this DRY, we can make one class with the repeating styles and then just add the ones that do not repeat in the individual ones.
Then in the HTML, we call both classes on each element
```suggestion

.projectpageimage {
  background-repeat: no-repeat;
  background-size: cover;
  display: flex;
  width: 100%;
  height: 206px;
  top: 500px;
  position: relative;
  border: 0.333333px solid rgba(235, 86, 86, 0.1);
}
.projectpageimage1 {
  background: url("Rectangle_21.svg");

}

.projectpageimage2 {
  background: url("second_project.svg");
}

.projectpageimage3 {
  background: url("thirdproject.svg");
}

.projectpageimage4 {
  background: url("fourthproject.svg");
}

.projectpageimage5 {
  background: url("fifthproject.svg");
}

.projectpageimage6 {
  background: url("sixthproject.svg");

}
```

Then in the HTML, in all places with those classes add
```
class= "projectpageimage projectpageimage1"
class= "projectpageimage projectpageimage2"
```
And so on for them all.

---

I have tried to do the same but it doesn't work. The repeat functionality gets lost if I do like this. And my elements get disturbed with respect to alignment. I will work on this in my next project. Since this does not impact my design functionality, I would like to proceed with this and in future, I will avoid use the above mentioned style. I will definitely work on it.